ctapipe/image/extractor.py

```diff
@@ -79,6 +79,7 @@ def sum_samples_around_peak(waveforms, peak_index, width, shift, ret):
 
 @njit([
     float64[:, :, :](float64[:, :, :], int64[:, :], int64),
```

Comment:
It would be better to switch the signatures to either:

```python
    float64[:, :, :](float64[:, :, :], int32[:, :], int64),
    float64[:, :, :](float32[:, :, :], int32[:, :], int64),
    float64[:, :, :](float64[:, :, :], int64[:, :], int64),
    float64[:, :, :](float32[:, :, :], int64[:, :], int64),
```

Waveforms can be float32 or float64 in my experience